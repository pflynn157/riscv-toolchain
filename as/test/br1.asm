beq x4, x5, LABEL
bne x4, x5, LABEL
blt x4, x5, LABEL
bge x4, x5, LABEL
bltu x4, x5, LABEL
bgeu x4, x5, LABEL
LABEL:
