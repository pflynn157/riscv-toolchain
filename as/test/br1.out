00520c63
00521a63
00524863
00525663
00526463
00527263
